arduino-ide-extension/src/browser/theia/messages/notifications-manager.ts

```diff
@@ -46,11 +45,4 @@ export class NotificationManager extends TheiaNotificationManager {
     }
     return Math.min((update.work.done / update.work.total) * 100, 100);
   }
-
-  /**
-   * For `public` visibility.
-   */
-  override getMessageId(message: Message): string {
-    return super.getMessageId(message);
-  }
```

Comment:
> anyway private method

Because it has `protected` visibility and accessing `protected` props this way, neither requires casting to `any` nor relaxing the `@typescript-eslint/dot-notation` ES-Lint rule with `{ "allowPrivateClassPropertyAccess": true }`.